js/components/authoring.js

```diff
@@ -133,10 +165,14 @@ export default class Authoring extends PureComponent {
     const doneClass = done ? 'done' : ''
     const activeClass = idx === step ? 'active' : ''
     return (
-      <span className={`label ${activeClass} ${doneClass}`}>{ info }</span>
+      <span className={`label ${activeClass} ${doneClass}`} key={'info' + idx}>{ info }</span>
     )
   }
 
+  renderDivider (idx) {
+    return <div className='divider' key={idx} />
```

Comment:
I realized my last PR left a duplicate key error because of the new children in an array. These key changes fix that